mipengine/controller/node_tasks_handler_celery.py

```diff
@@ -206,17 +292,21 @@ def get_remote_tables(self, context_id: str) -> List[str]:
         task_signature = self._celery_app.signature(
```

Comment:
`broker_connection_closed_handler` missing here?

Also, why have different decorators for exceptions? Why not unify `time_limit_exceeded_handler` with `broker_connection_closed_handler`? It won't be easy to forget as well and one place to add exception handling.

Also I don't know if you like it but there is a way to decorate all methods of a class with a decorator:
https://stackoverflow.com/questions/6307761/how-to-decorate-all-functions-of-a-class-without-typing-it-over-and-over-for-eac

Reply:
time_limit_exceeded_handler and broker_connection_closed_handler handle different situations, I do not want to tie them together